de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/account/XMPPAccount.java

```diff
@@ -1,6 +1,7 @@
 package de.fu_berlin.inf.dpp.account;
 
 import java.io.Serializable;
+import java.lang.Exception;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary import: java.lang classes are always implicitly imported. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=102&issues=AWP9mQNa3e-qcckj_oH-&open=AWP9mQNa3e-qcckj_oH-)

[ISSUE_KEY]: AWP9mQNa3e-qcckj_oH-